doc/changes/dev/13555.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bug where :func:`mne.io.read_raw_eyelink` raised an error when reading Eyelink files with an empty first recording block, by :newcontrib:`Varun Kasyap` (:gh:`13555`).
```

Comment:
content of the newcontrib role must match the entry in `names.inc` exactly (other than capitalization, which is ignored). See error:

> [towncrier-fragments]:14: ERROR: Indirect hyperlink target "new contributor Varun Kasyap"  refers to target "varun kasyap", which does not exist. [docutils]
[towncrier-fragments]:14: ERROR: Unknown target name: "varun kasyap". [docutils]

So either add `Pentamaraju` here, or remove it from `names.inc`